packages/babel-types/package.json

```diff
@@ -14,10 +14,11 @@
     "directory": "packages/babel-types"
   },
   "main": "lib/index.js",
+  "types": "lib/index-legacy.d.ts",
   "typesVersions": {
-    "<3.7": {
-      "lib/index.d.ts": [
-        "lib/index-ts3.7.d.ts"
+    ">=3.7": {
```

Comment:
should this be `<`

Reply:
No: `types` points to the `.d.ts` file for old TS versions. For new TS versions, (>= 3.7), we remap it back to `index.d.ts`.